tests/test_L102_S3_Hu_2007.py

```diff
@@ -1,15 +1,84 @@
 from . import ingest_tester
 from . import template
+import rdflib
 
-file_under_test = "L102_S3_Hu_2007"
 
-class IngestTestRunner(template.IngestTestSetup):
-    first_run = bool()
+class L102Test(template.IngestTestSetup):
     @classmethod
     def setUpClass(cls):
-        cls.file_under_test = file_under_test
+        cls.file_under_test = "L102_S3_Hu_2007"
         super().setUpClass()
 
-    def test_triples(self):
-         ingest_tester.print_triples(self)
+    def test_authors(self):
+        expected_authors = ["Hu, Tao", 
+                            "Juuti, Jari",
+                            "Vilkman, Taisto",
+                            "Jantunen, Heli"]
+        ingest_tester.test_authors(self, expected_authors)
+        ingest_tester.test_authors(self)
```

Comment:
You're requiring a subclass override to call a superclass method at its tail, which invites programmer errors. (Not a test-specific issue.)

The usual way to handle this is to
(1) Put the common test code in a separate, protected helper method (_test_authors or _test_authors_helper) in the superclass
(2) Have different superclass and subclass test_ methods e.g., test_authors_generic (superclass) and test_authors_specific (subclass) than use the helper method in both

If you want the superclass to REQUIRE the subclass to implement additional testing, then you can do that in one of two ways:

(1) Retrieve "expected_authors" from the subclass e.g., def _get_expected_authors(): raise NotImplementedError in the superclass test_authors, essentially an old-style abstract method without using the new abc module
(2) Similarly, delegate to def _test_authors(self): raise NotImplementedError, which should use the helper method above.